### ISSUE TITLE
~/.minetest filesystem override no longer works

### ISSUE THREAD

**Issue by eklmt** [flathub/net.minetest.Minetest#48]
Using an env override for MINETEST_USER_PATH doesn't seem to fix it, presumably because the value is hardcoded into the command, so any changes are overriden.

**Comment by Calinou** [comment]
cc @rubenwardy

**Comment by rubenwardy** [comment]
I don't know what you mean, please can you rephrase? Preferably with an example. Also, make sure you're on the most recent version

**Comment by eklmt** [comment]
The issue I'm having seems to be less to do with the override itself, and more that MINETEST_USER_PATH can't be overridden.

Here is my overrides file (~/.local/share/flatpak/overrides/net.minetest.Minetest):
```
[Context]
filesystems=~/.minetest

[Environment]
MINETEST_USER_PATH=$HOME/.minetest
```

The behaviour I was expecting was for this to make Minetest use ~/.minetest, but what happens instead is it will still create and use ~/.var/app/net.minetest.Minetest/.minetest.

As a  temporary workaround instead of the environment override, ```ln -s ../../../.minetest .var/app/net.minetest.Minetest/.minetest``` can be used to "redirect" it back to ~/.minetest (which also shows that the override portion of the config is indeed working).

Flatpak update shows no updates.
